### ISSUE TITLE
Page returned by jpa without content

### ISSUE THREAD

**Issue by fsantos92** [jpenren/thymeleaf-spring-data-dialect#15]
Hello,

When using pageable of spring mvc controller and my service return a Page without content(without data to show), the next button in pagination is enable. 
If I return new PageImpl<>(new ArrayList<>()), the pagination buttons stay disabled and there is a page 1 button.

Can you help me please?

**Comment by jpenren** [comment]
Excuse me for the delay, i will check this issue

**Comment by kanisso** [comment]
Hello jpenren..

Any news?

**Comment by jpenren** [comment]
Fixed in version 3.2.2. Please check it.

Note: you can hide all patinator if page content is empty with:
```
<div class="col-xs-12 col-md-6" th:if="${page.totalElements}">
    <nav>
..
    <nav>
</div>
```
